packages/trie/test/proof/range.spec.ts

```diff
@@ -481,4 +838,60 @@ describe('simple merkle range proofs generation and verification', () => {
 
     assert.equal(await verify(trie, entries, 0, entries.length - 1), false)
   })
+
+  it('passes randomly created tries with randomly selected ranges', async () => {
```

Comment:
I just resolved the other convo because I thought the copied test was solved, but it seems not. This test seems to appear twice? (and yes, I wrote it :smile: )